tracee/tracee.go

```diff
@@ -812,6 +812,41 @@ func (t *Tracee) shouldProcessEvent(e RawEvent) bool {
 
 func (t *Tracee) processEvent(ctx *context, args map[argTag]interface{}) error {
 	switch ctx.EventID {
+
+	//capture written files
+	case VfsWriteEventID, VfsWritevEventID:
+		if t.config.CaptureWrite {
+			filePath, ok := args[t.EncParamName[ctx.EventID%2]["pathname"]].(string)
+			if !ok {
+				return fmt.Errorf("error parsing security_bprm_check args")
+			}
+			// path should be absolute, except for e.g memfd_create files
+			if filePath == "" || filePath[0] != '/' {
+				return nil
+			}
+
+			destinationDirPath := filepath.Join(t.config.OutputPath, strconv.Itoa(int(ctx.MntID)))
+			if err := os.MkdirAll(destinationDirPath, 0755); err != nil {
+				return err
+			}
+			sourceFileStat, err := os.Stat(filePath)
```

Comment:
filePath cannot be Stat-ed as it can be in a different mount namespace (always true for containers)